docs/content/manual/release-specific/v1.4.0/test-csi-plugin-liveness-probe.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Test CSI plugin liveness probe
+---
+
+## Related discussion
+https://github.com/longhorn/longhorn/issues/3907
+
+## Reproduce Steps Before Fix
+
+**Given** healthy Longhorn cluster
+When delete the Longhorn CSI socket file on one of the node(`node-1`).
+    `rm /var/lib/kubelet/plugins/driver.longhorn.io/csi.sock`
+
+Then the `longhorn-csi-plugin-*` pod on `node-1` should restart.
+     the `csi-provisioner-*`     pod on `node-1` should restart.
+     the `csi-resizer-*`         pod on `node-1` should restart.
+     the `csi-snapshotter-*`     pod on `node-1` should restart.
+     the `csi-attacher-*`        pod on `node-1` should restart.
```

Comment:
Why will these sidecars be restarted as well? I thought the impact is only for the longhorn CSI plugin pod?